README.md

```diff
@@ -0,0 +1,34 @@
+# Rubicon
+
+A collection of tools to bridge between Python and other language
+environments.
+
+The core package doesn't do anything - it's just a namespace placeholder
+for the platform specific backends.
+
+# Related projects
+
+The following platform backends are available for Rubicon:
+
+- [rubicon-objc](https://github.com/pybee/rubicon-objc): for Objective C
+  (OS X and iOS)
+- [rubicon-java](https://github.com/pybee/rubicon-java): for Java
+
+# Community
+
+Rubicon is part of the [BeeWare suite](https://beeware.org). You can
+talk to the community through:
+
+- [@beeware@fosstodon.org on Mastodon](https://fosstodon.org/@beeware)
+- [Discord](https://beeware.org/bee/chat/)
+
+We foster a welcoming and respectful community as described in our
+[BeeWare Community Code of
+Conduct](https://beeware.org/community/behavior/).
+
+# Contributing
+
+If you experience problems with Rubicon, log them using the platform
+specific ticket trackers on GitHub
+([ObjC](https://github.com/pybee/rubicon-objc/issues),
```

Comment:
These urls should be on `beeware/rubicon-objc`

Reply:
done